e2e/react-start/transform-asset-urls/tests/transform-asset-urls.spec.ts

```
@@ -0,0 +1,195 @@
+import { expect, type Page } from '@playwright/test'
```

ESLint

[error] Member 'Page' of the import declaration should be sorted alphabetically.

(sort-imports)